cluster/common.sh

```diff
@@ -610,6 +610,8 @@ KUBERNETES_MASTER_NAME: $(yaml-quote ${KUBERNETES_MASTER_NAME})
 ALLOCATE_NODE_CIDRS: $(yaml-quote ${ALLOCATE_NODE_CIDRS:-false})
 ENABLE_CLUSTER_MONITORING: $(yaml-quote ${ENABLE_CLUSTER_MONITORING:-none})
 ENABLE_METRICS_SERVER: $(yaml-quote ${ENABLE_METRICS_SERVER:-false})
+ENABLE_METADATA_AGENT: $(yaml-quote ${ENABLE_METADATA_AGENT:-none})
```

Comment:
This sounds more like a boolean to me, should it be?

Reply:
I prefer to leave it in the same convention as ENABLE_CLUSTER_MONITORING, to support a case where similar agents would be needed for another monitoring system. This is also consistent with addon directory structure introduced in this PR: .../metadata-agent/stackdriver/ as opposed to just .../stackdriver-metadata-agent/